Produlytics/ui/back-end/src/main/resources/application.properties

```
@@ -1,6 +1,7 @@
-spring.datasource.url= jdbc:postgresql://db:5432/produlytics?currentSchema=public
-spring.datasource.jdbc-url= jdbc:postgresql://db:5432/produlytics?currentSchema=public
-spring.datasource.username= ui
-spring.datasource.password= ui
+spring.datasource.url= jdbc:postgresql://localhost:5432/produlytics?currentSchema=public
+spring.datasource.jdbc-url= jdbc:postgresql://localhost:5432/produlytics?currentSchema=public
+spring.datasource.username= postgres
+spring.datasource.password= password
+spring.jpa.show-sql= true
```

All'interno di docker l'ip del database è `db`, non `localhost`, e username e password per il back-end sono entrambi `ui`, non `postgres` e `password`.

```suggestion
spring.datasource.url=jdbc:postgresql://db:5432/produlytics?currentSchema=public
spring.datasource.jdbc-url=jdbc:postgresql://db:5432/produlytics?currentSchema=public
spring.datasource.username=ui
spring.datasource.password=ui
```